test/versioned/winston/winston.tap.js

```diff
@@ -162,6 +199,39 @@ tap.test('Winston instrumentation', { bail: true }, (t) => {
 
       logStuff({ logger, stream: jsonStream })
     })
+
+    t.test('should not double log nor instrument composed logger', (t) => {
+      const assertFn = (msg) => {
+        t.equal(msg['entity.name'], undefined, 'should not have entity name')
+        t.equal(msg['entity.type'], undefined, 'should not have entity type')
+        t.equal(msg.timestamp, undefined, 'should not have timestamp as number')
+        t.equal(msg.hostname, undefined, 'should not have hostname as string')
+        t.equal(msg.level, 'info')
+        t.equal(msg['trace.id'], undefined, 'msg should not have trace id')
+        t.equal(msg['span.id'], undefined, 'msg should not have span id')
+        t.ok(msg.message.includes('NR-LINKING'), 'should not contain NR-LINKING metadata')
```

Comment:
`t.ok` but `should not`?

Reply:
nice catch, i'll fix the other test that has this as well